app/src/main/res/values/strings.xml

```diff
@@ -16,6 +16,12 @@
     <string name="content_description_disable_private_browsing_button">Disable private browsing</string>
     <!-- Placeholder text shown in the search bar before a user enters text -->
     <string name="search_hint">Search or enter address</string>
+    <!-- Hint text shown when searching through history from the search bar -->
+    <string name="history_search_hint">Search History</string>
```

Comment:
Sorry, I think I made a mistake when creating the issue.  The second word should not be capitalized.  (ex: Search history)